### ISSUE TITLE
Metrics

### ISSUE THREAD

**Issue by lippytak** [codeforamerica/connect#14]
_Goal_: Make waiting on hold suck less (particularly for social service clients and their advocates).

In no particular order, I'd propose these as metrics:
1. # of calls initiated (usage growth)
2. total hours of hold time (usage growth)
3. # of unique users with 1+ call (user growth)
4. % of calls that reach a human (quality)

@daguar @alanjosephwilliams what do y'all think? Would these capture the main value of the project? Missing anything important?
